experimental/dds/tree2/src/test/feature-libraries/sequence-field/compose.spec.ts

```diff
@@ -123,15 +100,21 @@ describe("SequenceField - Compose", () => {
 	});
 
 	it("transient insert ○ modify", () => {
-		const transientDetach: ChangeAtomId = {
-			revision: tag2,
-			localId: brand(1),
-		};
 		const changes = TestChange.mint([], 42);
-		const insert = Mark.insert(2, brand(0), { transientDetach });
-		const modify = Mark.modify(changes, transientDetach);
-		const actual = compose([makeAnonChange([insert]), makeAnonChange([modify])], revInfos);
-		assert.deepEqual(actual, [insert]);
+		const insert = tagChange(Change.insert(0, 2), tag1);
+		const del = tagChange(Change.delete(0, 2, brand(1)), tag2);
+		const modify = tagChange(
+			Change.modifyDetached(0, changes, { revision: tag2, localId: brand(1) }),
+			tag3,
+		);
```

Comment:
We agreed that it would be nice to phase out the `Change.*` builders (which is low priority) so it would be nice not to introduce more usages of them.